cpp/include/cudf/types.h

```diff
@@ -156,6 +156,19 @@ typedef enum {
   GDF_NUM_COLORS, /** Add new colors above this line */
 } gdf_color;
 
+
+/* --------------------------------------------------------------------------*/
+/** 
+ * @Synopsis  These enums indicate how the nulls are treated in group_by/order_by operations.
+ */
+/* ----------------------------------------------------------------------------*/
+typedef enum {
+  GDF_NULL_AS_LARGEST = 0, 
+  GDF_NULL_AS_SMALLEST,
+  GDF_NULL_AS_LARGEST_FOR_MULTISORT
```

Comment:
Most of these are self-explanatory and don't really need in-line documentation, but I think this one does.